dbms/src/Storages/StorageDisaggregatedRemote.cpp

```diff
@@ -486,6 +489,13 @@ void StorageDisaggregated::buildRemoteSegmentInputStreams(
     pipeline.streams.reserve(num_streams);
 
     auto rs_operator = buildRSOperator(db_context, column_defines);
```

Comment:
https://github.com/pingcap/tiflash/blob/a5850b253e0cd1f494c9a5218213027aa157db9d/dbms/src/Storages/StorageDisaggregatedRemote.cpp#L429-L434

should add `table_scan.getPushedDownFilters()` to `query_info.dag_query`, like
```
query_info.dag_query = std::make_unique<DAGQueryInfo>(
            filter_conditions.conditions,
            table_scan.getPushedDownFilters(),
            analyzer->getPreparedSets(),
            analyzer->getCurrentInputColumns(),
            context.getTimezoneInfo());
```

Reply:
to use `table_scan.getPushedDownFilters()` to build rsoperator